demos/font/font_support_for_java/run_java_font_app.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+SCRIPT_NAME=run_java_font_app_internal.sh
+
+./create_java_font_app.sh
+
+docker run -it --device /dev/isgx --network host --rm -v `pwd`:`pwd` -w `pwd` occlum/occlum:0.15.1-ubuntu18.04 `pwd`/$SCRIPT_NAME
```

Comment:
We need to use the latest occlum build. The easy way is add this command in the readme file.

Reply:
Done